### PR TITLE
REACT-1.6 Create and initialize a React App Readme file added

### LINKED ISSUE #345: FRONTEND WITH FRAMEWORK: REACT-1.6 Create and initialize a React App
Contributors are required to include information about any prerequisite for installing React, its syntax , an example , an explanation (a good starting point https://reactjs.org/docs/create-a-new-react-app.html) about what each statement : For eg, What is npx, what does "cd" do , what does "npm start" do etc.

Also make sure -
Avoid copy/paste, We will be checking.
Changes should be made inside the Frontend_Web_Development_React_Angular_Vue/ dir inside Frontend_Web_Development_React_Angular_Vue branch.
Task will be assigned on first come first serve
Check out [Contribution Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md)
and [Code Of Conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by Adarshswain**
<hr>

## Description 📜
Created the documentation for the issue : .REACT-1.6 Create and initialize a React App
The Documentation includes How to create and intialize a react app with proper command to run using a terminal.
Fixes #345 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev React/Angular/Vue

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+ ```jsx
+ npx create-react-app girlscript-Winter-Of-contribution-app
+
+```
+ 
+## How This Command Works
+* The **npx** command will run a Node package without downloading it to your machine.
+*  The **create-react-app** script will install all of the dependencies and will build a base project in the **girlscript-Winter-Of-contribution-app** directory.
+
+* The code will download the dependencies and will create a base project. It may take a few minutes to finish. When it is complete, you will receive a **success** message. Your version may be slightly different if you use **yarn** instead of **npm**
```
Comment (mitraditya) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md:61]: Briefly mention what yarn is, in a line or two.

**Review by mitraditya — COMMENTED** [review]
Kindly make this change. Everything else looks fine.

File: Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md (hunk 1 above)
Comment (Adarshswain) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md:61]: ok sir I have to only write about yarn

Comment (mitraditya) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md:61]: Yes, since you've mentioned it there.

Comment (Adarshswain) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md:61]: ok sir nothing else I have to change

Comment (mitraditya) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md:61]: No, just change that part.

Comment (Adarshswain) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md:61]: bro, I will do it today only

Comment (Adarshswain) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md:61]: sir I have added the changes about yarn plz check it

File: Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+## What Is Yarn And How It Works
```
Comment (mitraditya) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md:63]: You do not have to create a separate subheading for yarn. Just mention the details in the same bullet point where you have mentioned "slightly different is you use yarn".

**Review by mitraditya — COMMENTED** [review]
Please edit this.

**Review by heet-kumar — CHANGES_REQUESTED** [review]
@Adarshswain try to add references at the bottom of the documentation .
References means from where you have referred and prepared this documentation give them credits too also.
Because it is mentioned in sever-announcement channel in discord that it is necessary to add references .

File: Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md (hunk 2 above)
Comment (Adarshswain) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md:63]: Ok sir I will change it

**Comment by Adarshswain** [comment]
ok sir I will do this changes

**Comment by Adarshswain** [comment]
sir I have done respective changes plz check

**Comment by Adarshswain** [comment]
sir i have added about yarn in same bulletin plz check also refernce

File: Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+ npx create-react-app girlscript-Winter-Of-contribution-app
+
+```
+ 
+## How This Command Works
+* The **npx** command will run a Node package without downloading it to your machine.
+*  The **create-react-app** script will install all of the dependencies and will build a base project in the **girlscript-Winter-Of-contribution-app** directory.
+
+* The code will download the dependencies and will create a base project. It may take a few minutes to finish. When it is complete, you will receive a **success** message. Your version may be slightly different if you use **yarn** instead of **npm**
+*  As Yarn is a package manager for our code that helps to manage project dependencies like NPM.
```
Comment (mitraditya) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md:62]: Add everything about yarn in the same bullet point. Also, there's a missing space between 'the' and 'yarn add command'. Kindly preview your file as to how it looks before making any update.

**Review by mitraditya — COMMENTED** [review]
Kindly make this change.

File: Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md (hunk 3 above)
Comment (Adarshswain) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md]: sir, I have changed it plz check it

Comment (mitraditya) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.6 Create and initialize a React App.md]: Looks good.

**Comment by Adarshswain** [comment]
sir, I have done all the change plz check it

**Review by anandhu720 — APPROVED** [review]

**Pull request merged**